Adyen/Core/Payment Methods/Abstract/AnyPaymentMethodDecoder.swift

```diff
@@ -273,3 +277,9 @@ private struct AtomePaymentMethodDecoder: PaymentMethodDecoder {
         .atome(try AtomePaymentMethod(from: decoder))
     }
 }
+
+private struct OnlineBankingPaymentMethodDecoder: PaymentMethodDecoder {
```

Comment:
We keep all decoders in `AnyPaymentMethodDecoder.swift`
Does it need to be here?

Reply:
I have added this in "AnyPaymentMethodDecoder.swift". Please check :)